src/main/java/com/google/devtools/build/lib/rules/cpp/CppConfiguration.java

```diff
@@ -1291,6 +1291,16 @@ public boolean hasSharedLinkOption() {
    * Returns the execution path to the linker binary to use for this build.
    * Relative paths are relative to the execution root.
    */
+  @SkylarkCallable(
+    name = "ld_executable",
+    structField = true,
+    doc = "Path to GNU binutils 'ld' binary."
```

Comment:
This is not true in all cases (gold, libtool, cl.exe), I'd say "Path to the linker binary" instead.